paper-form-fields.md

```diff
@@ -87,6 +91,55 @@ press the enter key. There's no default for this function.
 **escape()** You can pass a function called escape, that will be executed everytime
 you press the escape key. There's no default for this function.
 
-**format(value)** Gets the value and formats it, before updating the object.
-There's no default this function.
+**format(value)** Gets the value and formats it (only for displaying). There's no
+default function.
+
+**serialize(value)** Get the value and formats it before the actual value is
+updated. There's no default function.
+
+### check-box
+It renders a md-checkbox, which is actually _not_ a HTML input[type="checkbox"] element.
+This checkbox kan toggle a boolean and does not need any specific attributes. There're
+no attributes or methods to overwrite.
+
+### radio-group
+It renders a md-radio-group, including a md-radio-button for each given option. It does
+_not_ render an actual HTML input[type="radio"] element. There are no attributes or
+methods to override. Possible attributes and methods for this field include:
+
+**options (required)** This array can contain number, strings or objects. If you have an array
+objects, you need te pass a labelPath too.
+
+**labelPath** If the options contain objects, you have to pass a propertyPath as string
+to tell which attribute of the object should be displayed as the label of the option.
+
+### Auto-complete
+Renders a md-autocomplete that contains a ember-power-select.
+
+**multiSelect** If you want to select more than one option, you can pass true. It will
+render the 'paper-chips' component instead of the 'paper-autocomplete' component. The
+default is false.
+
+**enableSelectAll** You can display a 'select all' and 'deselect all' button by passing
+true. The default is false. Only works when 'multiSelect' is enabled.
+
+**allowClear** Wether the button to clear the input field should be available or not.
+Default is true.
+
+**readOnly** A boolean that can disable the autocomplete to make it a read only.
```

Comment:
Would it be better to have this `disabled` like all the other form-fields? Or are they `readOnly` now?

Reply:
I don't agree for several reasons:
1. We have `disabled` for **all** other form-fields, so why have this different?
2. I don't have any problem understanding `disabled=true`